nuxeo-drive-core/src/main/java/org/nuxeo/drive/hierarchy/userworkspace/adapter/UserWorkspaceTopLevelFolderItem.java

```diff
@@ -81,6 +81,8 @@ public UserWorkspaceTopLevelFolderItem(String factoryName, DocumentModel userWor
         canScrollDescendants = false;
         this.userWorkspace = userWorkspace;
         this.syncRootParentFactoryName = syncRootParentFactoryName;
+        // detach user workspace as we will use another session to update it
+        userWorkspace.detach(true);
```

Comment:
Could that be linked to https://jira.nuxeo.com/browse/NXP-21496? In which case might be worth mentioning.